src/main/java/org/isf/accounting/rest/BillController.java

```diff
@@ -111,7 +111,7 @@ public ResponseEntity<FullBillDTO> newBill(@RequestBody FullBillDTO newBillDto)
 
 		Bill bill = billMapper.map2Model(newBillDto.getBill());
 
-		Patient pat = patientManager.getPatientByName(bill.getPatName()); // FIXME: verify why we were searching by name
+		Patient pat = patientManager.getPatientById(bill.getBillPatient().getCode());
```

Comment:
@ArnaudFonzam can you please clarify why we were searching by name instead of by id? Can we safely remove this?

Reply:
@mwithi  yes we can change and it is  on controller and the ui is not available yet. it is better to search by id.

Reply:
Ok, thanks!